circle.yml

```diff
@@ -4,6 +4,7 @@ machine:
     version: oraclejdk8
   environment:
     TERM: dumb  # don't let gradle use fancy ansi seq code
+    BUILD_IMAGE: danieln/digdag-build:20160823085939Z-4c3c94d6925e4e476fca04260ec6be6352351de2
```

Comment:
this image is built using `docker/push.sh` command, right? it would be great if here has a comment that says about it.